src/sentry/tasks/summaries/weekly_reports.py

```diff
@@ -164,7 +164,10 @@ def prepare_organization_report(
 
             project_ctx = cast(ProjectContext, ctx.projects_context_map[project.id])
             if key_errors:
-                project_ctx.key_errors = [(e["group_id"], e["count()"]) for e in key_errors]
+                if features.has("organizations:snql-join", project.organization):
+                    project_ctx.key_errors = [(e["e.group_id"], e["count()"]) for e in key_errors]
+                else:
+                    project_ctx.key_errors = [(e["group_id"], e["count()"]) for e in key_errors]
```

Comment:
is the only difference `e.group_id` vs `group_id`? i would consider having this conditional statement setting a variable called `group_id_alias`, and then only one statement below it that sets `project_ctx.key_errors`. as of now i found it hard figuring out what the difference was between the two statements

Reply:
could also just add a comment instead.